### PR TITLE
OP-19199: implemented the DB Schema version to verify in post DB upgrade run

### PR DESCRIPTION
**Pull request by sudhakaropsmx**
https://devopsmx.atlassian.net/browse/OP-19199

### CODE REVIEW

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py [hunk 1]
```diff
@@ -154,7 +159,7 @@ def perform_migration(version):     # post-upgrade Data Migration (to be run as
         print("Updating application name in audit events table")
         updateApprovalGateAudit()
 
-        commit_transactions()
+        #commit_transactions()
```
Comment (Pranav-b-7) [inline scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py:162]: why this line is commented ?

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py [hunk 2]
```diff
@@ -1525,7 +1530,19 @@ def addDBVersion(version):
...
+    try:
+        # fetch db_version from platform db
+        cur_platform.execute("select version_no from db_version order by updated_at desc limit 1")
+        db_version = cur_platform.fetchone()
+        if db_version is not None and db_version[0] != version:
+           raise Exception("Post upgrade could not proceed, upgrade version is not same existing schema")
+        else:
+           logging.info("Failed fetch the ISD Schema version from Platform DB")       
+    except Exception as e:
+        print("Exception occurred while adding db version : ", e)
```
Comment (Pranav-b-7) [inline scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py:1543]: it should be exception occurred while fetching db version

**Review by Pranav-b-7 — CHANGES_REQUESTED** [review]

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py (hunk 1 above)
Comment (sudhakaropsmx) [reply scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py]: missed to remove commented while commit. now resolved

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py (hunk 2 above)
Comment (sudhakaropsmx) [reply scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py]: updated the Exception messages

**Review by Pranav-b-7 — APPROVED** [review]

**Pull request merged**